README.md

```diff
@@ -468,6 +517,14 @@ On Windows, Disable `git core.autocrlf` (One Time Only)
 git config --global core.autocrlf false
 ```
 
+### CV Manager Common Issues
+
+1. After logging into the webapp, you are presented with a page reading "Login Unsuccessful: Unknown Error Occurred"
```

Comment:
suggestion (non-blocking): it would be nice to expand this section a bit and include other things like IP change issues and needing to rebuild the web